android/dosidicus_mobile/ui/app.py

```
@@ -277,6 +294,36 @@ def _add_decoration(self, deco_type, category):
+            return
+        try:
+            accelerometer.enable()
+        except Exception as e:
+            print(f"[Dosidicus] accelerometer unavailable: {e}")
+            return
+        Clock.schedule_interval(self._poll_shake, 0.1)
+
+    def _poll_shake(self, dt):
+        try:
+            val = accelerometer.acceleration
+        except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)